c2cgeoportal/scaffolds/create/mapserver/c2cgeoportal.map.mako_tmpl

```diff
@@ -0,0 +1,93 @@
+#
+# MapServer Mapfile
+#
+# Test requests:
+#
+# WMS GetCapabilities:
+# /mapserv?service=wms&version=1.1.1&request=getcapabilities
+#
+# WMS GetMap:
+# /mapserv?service=wms&version=1.1.1&request=getmap&bbox=-180,-90,180,90&layers=countries&width=600&height=400&srs=EPSG:4326&format=image/png
+#
+# WMS GetFeatureInfo:
+# /mapserv?service=wms&version=1.1.1&request=getfeatureinfo&bbox=-180,-90,180,90&layers=countries&query_layers=countries&width=600&height=400&srs=EPSG:4326&format=image/png&x=180&y=90&info_format=application/vnd.ogc.gml
+#
+
+MAP
+    NAME "geoportal"
+
+    # For Windows users: uncomment this line and adapt it to your
+    # own mapserver's nad folder (use regular slash "/")
+    # CONFIG "PROJ_LIB" "C:/path/to/ms4w/proj/nad"
+
+#    EXTENT 420000 30000 900000 350000 ## should be defined for better performance
```

Comment:
hey, this is almost what I did in my local project. I will look at this tomorrow.